code/components/voip-mumble/src/MumbleAudioInput.cpp

```diff
@@ -417,14 +428,20 @@ void MumbleAudioInput::SendQueuedOpusPackets()
 
 		const auto& [packet, frames] = packetChunk;
 
-		char outBuf[16384];
+		// Maximum size of Opus audio frames is 8191, but we *can't* actually send that much, FiveMs mumble will drop the player, while other implementations will just ignore the packet.
+		// This should be split into multiple packets for higher voice bitrates.
+		// https://mumble-protocol.readthedocs.io/en/latest/voice_data.html#packet-format 
+		// https://mumble-protocol.readthedocs.io/en/latest/voice_data.html#opus-audio-frames
+		static char outBuf[kMaxUdpPacket] = {};

```

Comment:
Probably shouldn't make this static (and remove the zero-initializer)

Reply:
Is there any reason not to here, anything used by these are copied out

1. The usage here is initialize buffer
2. Use buffer for `PacketDataStream`
3. Append packet to backing buffer
4. Send to `SendVoice` which will call `SendUDP` (or `Send` for TCP tunnel if UDP isn't available), 

5a. if its send via UDP it will be copied out into a new `outBuf` when its encrypted
5b. if its send via TCP it will be encrypted via TLS (which will block until encryption is done)

This all happens sequentially so there isn't any risk of this being used by multiple threads.